src/app/owners/owner.service.spec.ts

```diff
@@ -87,33 +95,117 @@ describe('OnwerService', () => {
     httpTestingController.verify();
   });
 
-  /// OwnerService method tests begin ///
-
-  describe('#getOwners', () => {
-    let expectedOwners: Owner[];
+  it("should return expected owners (called once)", () => {
```

Comment:
Did you have use the linter (`ng lint`) ? I remember single quote should be used.